graph/tags.go

```diff
@@ -81,6 +82,7 @@ func NewTagStore(path string, cfg *TagStoreConfig) (*TagStore, error) {
 	}
 	// Load the json file if it exists, otherwise create it.
 	if err := store.reload(); os.IsNotExist(err) {
+		logrus.Debugf("%s", err)
```

Comment:
@LK4D4 Previously, when the tag store file is not available there is no proper message in logs (whether the tag store file is missing or empty) on daemon startup. 

Therefore adding this line would print a message when the `/var/lib/repositores-{aufs,vfs}` file is not available like follows: 
```
DEBU[0000] open /var/lib/docker/repositories-aufs: no such file or directory
``` 
I will change this to be more meaningful to than just printing the error `logrus.Debugf("Tag store file not found: %s", err)` Let me know its ok. Thanks